Form/FOSUBRegistrationFormHandler.php

```diff
@@ -88,18 +84,17 @@ public function process(Request $request, Form $form, UserResponseInterface $use
             return $processed;
         }
 
+        if (!$request->isMethod('POST')) {
+            return false;
```

Comment:
This is wrong as initializing the form data must be done for the GET request too (otherwise the form will be empty)